app/src/page/rewards/view.js

```diff
@@ -106,25 +107,15 @@ class RewardsPage extends React.PureComponent {
           <Text style={rewardStyle.infoText}>This app is unable to earn rewards due to an authentication failure.</Text>
         </View>
       );
-    } else if (!rewards || rewards.length <= 0) {
-      return (
-        <View style={rewardStyle.busyContainer}>
-          <Text style={rewardStyle.infoText}>
-            {(claimed && claimed.length) ?
-              "You have claimed all available rewards! We're regularly adding more so be sure to check back later." :
-              "There are no rewards available at this time, please check back later."}
-          </Text>
```

Comment:
You could keep this logic and render the reward code below it. Then users still have some idea that more rewards will be generated. That is what the desktop app does.

Reply:
https://github.com/lbryio/lbry-desktop/blob/master/src/renderer/page/rewards/view.jsx#L127-L144

Reply:
I thought of doing this but decided to go with just showing the custom reward code, because there's limited screen real estate on mobile devices. All new unclaimed rewards will automatically show up at the top, so it's not really an issue.